bugwarrior/notifications.py

```diff
@@ -89,23 +94,7 @@ def send_notification(issue, op, conf):
             priority=1,
         )
         return
-    elif notify_backend == 'pynotify':
-        _cache_logo()
-
-        import pynotify
-        pynotify.init("bugwarrior")
-
-        if op == 'bw finished':
-            message = "Finished querying for new issues.\n%s" %\
-                issue['description']
-        else:
-            message = "%s task: %s" % (op, issue['description'])
-            metadata = _get_metadata(issue)
-            if metadata is not None:
-                message += metadata
-
-        pynotify.Notification("Bugwarrior", message, logo_path).show()
-    elif notify_backend == 'gobject':
+    elif notify_backend == 'gobject' or notify_backend == 'pynotify':
```

Comment:
nitpick: Would it be better to set `notify_backend = 'gobject'` after the warning so this deprecated setting is handled in just one place? Seems simpler to cleanup if and when it's removed and less confusing if references to historical features are all in one place.